client/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
```

Comment:
I recommend using styled-components with macro. Then the rendered styles in the HTML got the component name attached.
```suggestion
import styled from 'styled-components/macro';
```